build.sbt

```diff
@@ -52,7 +52,6 @@ lazy val demo =
       ),
       webpackMonitoredDirectories += (Compile / resourceDirectory).value,
       webpackResources                := (baseDirectory.value / "webpack") * "*.js",
-      Compile / includeFilter         := "*",
```

Comment:
Is this important?

Reply:
I seem to have mistakenly ported `includeFilter in webpackMonitoredFiles := "*"` to this when migrating to the new sbt syntax: `https://github.com/gemini-hlsw/lucuma-ui/commit/012e635b5a843bbddbdab96298f17a965bc108c5#diff-5634c415cd8c8504fdb973a3ed092300b43c4b8fc1e184f7249eb29a55511f91L59`.

Maybe it should be `webpackMonitoredFiles / includeFilter := "*"` ?

Reply:
Makes sense, done.